sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -522,6 +522,14 @@ def test_replace_map():
     assert N == K
 
 
+def test_rot90():
+    A = Matrix([[1, 2], [3, 4]])
+    assert A == A.rot90(0)
+    assert A.rot90(1) == A.rot90(-3)
+    assert A.rot90(2) == A.rot90(-2)
+    assert A.rot90(3) == A.rot90(-1)
+    assert A.rot90() == A.rot90(-7) == A.rot90(-3) == Matrix(((3, 1), (4, 2)))
```

Comment:
The cases for k=2 and k=3 need similar tests since otherwise they could be failing silently with the current test. Do you also see why `assert A.rot90(1) == A.rot90(-3)` is redundant with the test that you added?

Reply:
yeah that 😅